app/javascript/articles/__tests__/__snapshots__/ArticleLoading.test.jsx.snap

```diff
@@ -2,41 +2,35 @@
 
 exports[`<LoadingArticle /> should render 1`] = `
 <div
-  class="single-article single-article-small-pic big-article"
-  style={
-    Object {
-      "paddingBottom": "0",
-    }
-  }
+  class="crayons-story"
```

Comment:
General question about CSS class naming @ludwiczakpawel. I thought we were only using the `crayons-` prefix for design system components.

Reply:
Good question, I was wondering the same but maybe you've already talked about this?

Reply:
No, waiting for input from @ludwiczakpawel

Reply:
not really sure what should be the answer here. on one hand we could consider feed card as a standalone component. on the other hand i know it's quite a beast to be considered a component.

i think i'll be using `crayons-` prefix not only for components but generally for crayons-related stuff, like for example classes defining views/layouts (view with 2 sidebars, view with no sidebar, etc)... so why not also for bigger components like feed cards?

but i'm not sold on any of these approaches.. what are your thoughts

Reply:
This is not a blocker for this PR.

I guess my point is, the feed card (`<Article />` component) is an application component, so not part of the design system. It's a component that consumes the design system components. I can see there being a crayons `<Card />` component that could be used as the container of an `<Article />` component. Otherwise, I don't see the point to prefixing CSS classes with `crayons-` for stuff that isn't really design system specific. We can discuss further in Slack.

![Woman holding a taco saying "Let's taco 'bout it!](https://media.giphy.com/media/WqLmcthJ7AgQKwYJbb/giphy.gif)